airbyte-integrations/connectors/source-twilio/source_twilio/schemas/conversation_participants.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "account_sid": {
+      "type": ["null", "string"]
+    },
+    "conversation_sid": {
+      "type": ["null", "string"]
+    },
+    "sid": {
+      "type": ["null", "string"]
+    },
+    "identity": {
+      "type": ["null", "string", "integer"]
+    },
+    "attributes": {
+      "type": "object",
+      "properties": {
+        "role": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "messaging_binding": {
+      "type": "object",
```

Comment:
Are you going to always expect a message binding? Maybe is better ["null", "object"]